crates/solvers/src/infra/config/legacy/file.rs

```diff
@@ -0,0 +1,50 @@
+use {
+    crate::{domain::eth, infra::contracts, util::serialize},
+    reqwest::Url,
+    serde::Deserialize,
+    serde_with::serde_as,
+    std::path::Path,
+    tokio::fs,
+};
+
+#[serde_as]
+#[derive(Deserialize)]
+#[serde(rename_all = "kebab-case", deny_unknown_fields)]
+struct Config {
+    /// Chain id used to automatically determine the address
+    /// of the WETH contract and for metrics.
+    #[serde_as(as = "serialize::ChainId")]
+    chain_id: eth::ChainId,
+
+    /// The solver name used in metrics.
+    pub solver_name: String,
+
+    /// The URL of the solver.
+    pub base_url: String,
+
+    /// Controls value of the `max_nr_exec_orders` parameter.
+    /// Uses 100 as the default value.
+    pub max_nr_exec_orders: Option<u32>,
```

Comment:
This is a legacy Quasimodo thing, I would get rid of it and use some hard coded value in the `boundary/legacy` module instead of making it configuratble.